src/app/mauro/research-plugin.service.ts

```diff
@@ -75,4 +81,13 @@ export class ResearchPluginService {
       .theme()
       .pipe(map((response: MdmIndexResponse<KeyValueIdentifier>) => response.body.items));
   }
+
+  listSharedDataSpecifications(): Observable<DataSpecification[]> {
+    return this.endpoints.pluginResearch.listSharedDataSpecifications().pipe(
+      switchMap((response: DataModelIndexResponse): Observable<DataModel[]> => {
```

Comment:
I think this could be simplified...

```ts
return this.endpoints.pluginResearch.listSharedDataSpecifications().pipe(
  map((response: DataModelIndexResponse) => response.body.items.map(mapToDataSpecification))
)
```